improver/wxcode/utilities.py

```diff
@@ -95,7 +95,7 @@
 DAYNIGHT_CODES = [1, 3, 10, 14, 17, 20, 23, 26, 29]
 
 
-def update_tree_units(tree: Dict) -> Dict:
+def update_tree_units(tree: Dict[str, Dict[str, Any]]) -> Dict:
```

Comment:
Should this be as shown below to include the type of the output?
```suggestion
def update_tree_units(tree: Dict[str, Dict[str, Any]]) -> Dict[str, Dict[str, Any]]:
```